x-pack/plugins/secops/public/store/local/errors/actions.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import actionCreatorFactory from 'typescript-fsa';
+
+const actionCreator = actionCreatorFactory('x-pack/secops/local/hosts');
+
+export const addError = actionCreator<{ id: string; title: string; message: string }>(
+  'UPDATE_ERRORS'
```

Comment:
Should this be `ADD_ERROR` since you only ever [concat new errors](https://github.com/elastic/kibana/pull/30332/files#diff-ebce825713614964b39040a570d32a6eR17) and not update existing? Or is this redux nomenclature?

Reply:
Ahh, I see, the action matches the name of the structure saved in redux, which is `errors`.

Reply:
It should be `ADD_ERRORS` so I updated it to that.